Internship1/Controllers/MessageDefinitionController.cs

```diff
@@ -19,10 +19,49 @@ public class MessageDefinitionController : Controller
         // GET: MessageDefinition
         public ActionResult Index()
         {
-             List<MessageDefinition> list = Import.ImportFile();
-             ViewBag.json = ExportJson.ExportJSON(list);
-            
-             return View();
+            List<ExportModel> en = Import.ImportFileEn(@"C:\Users\Korisnik\Desktop\Posao\Internship - Execom\MessageDefinition.csv");
+            List<ExportModel> de = Import.ImportFileDe(@"C:\Users\Korisnik\Desktop\Posao\Internship - Execom\MessageDefinition.csv");
+            List<ExportModel> es = Import.ImportFileEs(@"C:\Users\Korisnik\Desktop\Posao\Internship - Execom\MessageDefinition.csv");
+            List<ExportModel> fr = Import.ImportFileFr(@"C:\Users\Korisnik\Desktop\Posao\Internship - Execom\MessageDefinition.csv");
+            List<ExportModel> it = Import.ImportFileIt(@"C:\Users\Korisnik\Desktop\Posao\Internship - Execom\MessageDefinition.csv");
+            List<ExportModel> ja = Import.ImportFileJa(@"C:\Users\Korisnik\Desktop\Posao\Internship - Execom\MessageDefinition.csv");
+            List<ExportModel> pt = Import.ImportFilePt(@"C:\Users\Korisnik\Desktop\Posao\Internship - Execom\MessageDefinition.csv");
+            List<ExportModel> ru = Import.ImportFileRu(@"C:\Users\Korisnik\Desktop\Posao\Internship - Execom\MessageDefinition.csv");
+            List<ExportModel> zh = Import.ImportFileZh(@"C:\Users\Korisnik\Desktop\Posao\Internship - Execom\MessageDefinition.csv");
```

Comment:
No need to have import for each language, this should be generic so that the Import service automatically create all this lists